code/controllers/configuration.dm

```diff
@@ -280,6 +280,7 @@ var/list/gamemode_cache = list()
 
 	var/iterative_explosives_z_threshold = 10
 	var/iterative_explosives_z_multiplier = 0.75
+	var/iterative_explosives_z_subtraction = 2

```

Comment:
default value of 2 reduces the explosion, nerfing them. Which is not documented in changelog or PR description.

Reply:
This can be overridden by updating the configuration. Whoever controls the configuration is expected to change it to their desired value.

Reply:
This should be 0 imho